tests/integration/post.hurl

```diff
@@ -0,0 +1,35 @@
+# Test POST /users endpoint
+
+# 1. Create new user (should succeed)
+POST http://localhost:8080/v1/users
+[BasicAuth]
+user1@email.com:password1
+HTTP 201
+[Asserts]
+jsonpath "$.status" == "ok"
+
+# 2. Create same user again (should succeed with 201)
```

Comment:
If the user already exists, the endpoint should return a 200. 201 means a resource was created.